uiowa/uiowa
# Feature: admissions hotfix for aos teaser and taxonomy terms

This check (https://github.com/uiowa/uiowa/pull/3555/files#diff-8b86d0481fbe21d2b3f16cde4ff1be44614d90bdd3154d9ea894b87fae016edbR263) was adding `card-media--right` to every teaser template as you can see it being applied on the cards here: https://admissions.prod.drupal.uiowa.edu/academics/health-sci.  

This pr:
-  Updates the check into a switch statement and it appears add the classes for articles and pages but is no longer adding the class to areas of study teasers. 
- Adds a `logged_in` check on the menu local task hook so that the "View" tab does not appear for anonymous users. 

# How to test
- `blt ds --site=admissions.uiowa.edu`
- Compare https://admissions.prod.drupal.uiowa.edu/academics/health-sci against https://admissions.local.drupal.uiowa.edu/academics/health-sci and verify the cards are not messed up. 
- Add a featured content list to a page containing articles and pages that contain featured images and verify that each card is getting the `card-media--right` class. 
- Review the code updates in "Files changed"
- Compare https://admissions.local.drupal.uiowa.edu/colleges/college-education with https://admissions.prod.drupal.uiowa.edu/colleges/college-education and verify that the local task menu "View" link no longer appears if logged out. 
- While logged in, verify that the local task editor icons appear on a page like https://admissions.local.drupal.uiowa.edu/colleges/college-education.
